components/UserProvider.js

```diff
@@ -82,7 +82,7 @@ class UserProvider extends React.Component {
   logout = async () => {
     removeFromLocalStorage(LOCAL_STORAGE_KEYS.ACCESS_TOKEN);
     this.setState({ LoggedInUser: null, errorLoggedInUser: null });
-    await this.props.client.resetStore();
+    await this.props.client.clearStore();
```

Comment:
This will prevent queries from being refetched, which I assume could have side-effects in various places. Can you clarify why it's needed?